numpy/core/tests/test_overrides.py

```diff
@@ -558,18 +558,19 @@ def test_array_like_fromfile(self, numpy_ref):
 
         data = np.random.random(5)
 
-        fname = tempfile.mkstemp()[1]
-        data.tofile(fname)
-
-        array_like = np.fromfile(fname, like=ref)
-        if numpy_ref is True:
-            assert type(array_like) is np.ndarray
-            np_res = np.fromfile(fname, like=ref)
-            assert_equal(np_res, data)
-            assert_equal(array_like, np_res)
-        else:
-            assert type(array_like) is self.MyArray
-            assert array_like.function is self.MyArray.fromfile
+        with tempfile.NamedTemporaryFile(suffix=".txt") as tmp:
+            fname = tmp.name
+            data.tofile(fname)
```

Comment:
This is causing problems on windows. Try `data.tofile(tmp)`. Windows only allows a single process to access an open file and I suspect that is the problem.

Reply:
Thanks!  Well, I tried with `TemporaryDirectory` and just fixed the actual filename as a file inside that directory... That seems to work fine, lets see what windows thinks.
Maybe using `tmp` directly is nicer, but then I have to deal with `seek(0)` also...